bedrock/firefox/tests/test_views.py

```diff
@@ -654,66 +654,66 @@ def test_get_context_data_one_star(self):
 @override_settings(DEV=False)
 @patch('bedrock.firefox.views.l10n_utils.render')
 class TestFirefoxFeatures(TestCase):
-    def test_new_template(self, render_mock):
+    def test_en_us(self, render_mock):
         req = RequestFactory().get('/firefox/features/')
         req.locale = 'en-US'
         views.features_landing(req)
         render_mock.assert_called_once_with(req, 'firefox/features/index.html')
 
     @patch.object(views, 'lang_file_is_active', lambda *x: False)
-    def test_old_template(self, render_mock):
+    def test_non_en_us(self, render_mock):
         req = RequestFactory().get('/firefox/features/')
         req.locale = 'de'
         views.features_landing(req)
-        render_mock.assert_called_once_with(req, 'firefox/features.html')
+        render_mock.assert_called_once_with(req, 'firefox/features/index.html')
 
 
 @override_settings(DEV=False)
 @patch('bedrock.mozorg.views.l10n_utils.render')
 class TestFirefoxProductDesktopView(TestCase):
-    def test_new_template(self, render_mock):
+    def test_en_us(self, render_mock):
         view = views.FirefoxProductDesktopView()
         view.request = RequestFactory().get('/firefox/desktop/')
         view.request.locale = 'en-US'
         eq_(view.get_template_names(), ['firefox/products/desktop.html'])
 
     @patch.object(views, 'lang_file_is_active', lambda *x: False)
-    def test_old_template(self, render_mock):
+    def test_non_en_us(self, render_mock):
         view = views.FirefoxProductDesktopView()
         view.request = RequestFactory().get('/firefox/desktop/')
         view.request.locale = 'fr'
-        eq_(view.get_template_names(), ['firefox/desktop/index.html'])
```

Comment:
Incidentally, if we do remove those pages - would we also be free to remove the old family nav itself?